components/slack/sources/new-reaction-added/new-reaction-added.mjs

```diff
@@ -44,14 +44,25 @@ export default {
         "ignoreBot",
       ],
     },
+    iconEmoji: {
+      propDefinition: [
+        common.props.slack,
+        "icon_emoji",
+      ],
+      description: "Provide an emoji to use as filter the events. E.g. `fire`",
+      optional: false,
+    },
   },
   methods: {
     ...common.methods,
     getSummary() {
       return "New reaction added";
     },
     async processEvent(event) {
-      if ((this.ignoreBot) && (event.subtype == "bot_message" || event.bot_id)) {
+      if (
+        ((this.ignoreBot) && (event.subtype == "bot_message" || event.bot_id)) ||
+        (this.iconEmoji && event.reaction !== this.iconEmoji)
```

Comment:
With `this.iconEmoji` as an array, you could check if it includes the event reaction.
```suggestion
        (this.iconEmoji?.length > 0 && !this.iconEmoji.includes(event.reaction))
```